helm/prometheus-rules/templates/alerting-rules/inhibit.prometheus-agent.rules.yml

```
@@ -11,15 +11,14 @@ spec:
   - name: inhibit.prometheus-agent
     rules:
     # this inhibition fires when a cluster is not running prometheus-agent.
-    # we retrieve the list of existing cluster IDs from `kube_namespace_created`
-    # then compare it with the list of deployed observability-bundles from `app_operator_app_info`
+    # we retrieve the list of existing cluster IDs from `prometheus_build_info` with app="prometheus"
+    # then compare it with the list of `prometheus_build_info` with instance="prometheus-agent"
```

I think now the syntax has been simplified, we could also simplify the explanation:
```suggestion
    # If we we find prometheus_build_info for instance="prometheus-agent", it means prometheus-agent is running on this cluster
    # so, raise an inhibition unless prometheus-agent runs on the cluster 
```

but, do we want to inhibit when `prometheus-agent` runs on the cluster (current inhibition) or when `prometheus-agent` **is supposed to** run on the cluster (previous version)?

---

we just want to inhibit if the cluster is not running prometheus-agent